app/src/main/res/layout/fragment_email_verification_interstitial.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/white"
+    android:padding="@dimen/grid_4">
+
+    <ImageView
```

Comment:
add `android:contentDescription="@null"` if it's not important for accessibility